libgalois/src/GraphTopology.cpp

```diff
@@ -569,6 +583,26 @@ katana::ProjectedTopology::MakeTypeProjectedTopology(
     }
   });
 
+  katana::do_all(
```

Comment:
I don't like this code at all. We need to either use `std::bitset` or `vector<bool>` (both of which I believe pack bits into bytes). I don't like the divide by 8 here, multiply by 8 there, and take a remainder over there. This is all recipe for future subtle bugs. If none of the STL classes fit the bill, we should write or modify our own class, e.g. DynamicBitset. cc: @roshandathathri

Reply:
I am not sure of how to do this as I also want to be able to convert the data to the format of `const uint8_t*`, which I am currently able to do with the help of `NUMAArray`. But yes if there is a solution that allows parallel setting of bits plus also a conversion to` const uint8_t*`, then it's fine. thank you

Reply:
ok, for now, you can at least refactor this code into a common function and then put a TODO. You're doing the same thing twice for nodes and edges.